spec/services/metrics/weekly/review_turaround/per_user_project_spec.rb

```diff
@@ -0,0 +1,61 @@
+require 'rails_helper'
+
+RSpec.describe Metrics::Weekly::ReviewTurnaround::PerUserProject do
+  describe '.call' do
+    let(:project) { create(:project) }
+    let(:users_project) { create(:users_project, project: project) }
+
+    context 'generating a weekly metric for a user who has made his first review' do
+      let!(:metric) { create(:metric, ownable: users_project, value_timestamp: Time.zone.today) }
+      let(:weekly_metric_just_created) { Metric.find_by(interval: :weekly) }
+      it 'creates the metric with the same value the current daily metric' do
+        described_class.call
+        expect(metric.value).to eq(weekly_metric_just_created.value)
+      end
+
+      it 'creates just one weekly metric per daily metric' do
```

Comment:
one weekly metric per daily metric? shouldn't it be one weekly metric per seven daily metrics?

Reply:
but in the context when the user has made his first review, in that case the value of the daily metric is the same as weekly metric and just one weekly metric is generated per daily metric generated in that day